{
	"id": "oval:org.opensuse.security:def:202445230",
	"version": "1",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2024-45230",
		"affected": {
			"family": "unix",
			"platform": [
				"openSUSE Tumbleweed"
			]
		},
		"reference": [
			{
				"ref_id": "Mitre CVE-2024-45230",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45230",
				"source": "CVE"
			},
			{
				"ref_id": "SUSE CVE-2024-45230",
				"ref_url": "https://www.suse.com/security/cve/CVE-2024-45230",
				"source": "SUSE CVE"
			},
			{
				"ref_id": "SUSE-SU-2024:3139-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2024-September/019372.html",
				"source": "SUSE-SU"
			}
		],
		"description": "\n    ** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided.\n    ",
		"advisory": {
			"from": "security@suse.de",
			"severity": "Important",
			"cve": [
				{
					"text": "CVE-2024-45230 at SUSE",
					"href": "https://www.suse.com/security/cve/CVE-2024-45230/",
					"impact": "high",
					"cvss3": "7.5/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
				},
				{
					"text": "CVE-2024-45230 at NVD",
					"href": "https://nvd.nist.gov/vuln/detail/CVE-2024-45230"
				}
			],
			"bugzilla": [
				{
					"text": "SUSE bug 1229823",
					"href": "https://bugzilla.suse.com/1229823"
				}
			],
			"issued": {
				"date": "2024-09-05"
			},
			"updated": {
				"date": "2024-09-05"
			},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/o:opensuse:tumbleweed"
				]
			}
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:org.opensuse.security:tst:2009941108",
						"comment": "python310-Django4-4.2.16-1.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009941109",
						"comment": "python311-Django4-4.2.16-1.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009941110",
						"comment": "python312-Django4-4.2.16-1.1 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.opensuse.security:tst:2009634834",
				"comment": "openSUSE Tumbleweed is installed"
			}
		]
	}
}
